packages/widget-chat/src/components/list-separator/index.js

```diff
@@ -0,0 +1,29 @@
+import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+
+import styles from './styles.css';
+
+function ListSeparator(props) {
+  const mainStyles = [`separator`, styles.separator];
+  const textStyles = [`separator-text`, styles.separatorText];
+  if (props.classNames) {
+    props.classNames.forEach((className) => {
+      mainStyles.push(className);
+      mainStyles.push(styles[className]);
+      textStyles.push(className);
+      textStyles.push(styles[className]);
+    });
```

Comment:
Good call. I was trying to model it after the web client's components but we can definitely divert from it and do it that way. I'm not a big fan of it either, so I'm convinced 😄